packages/app/src/device-info/services/device-info.service.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Result, ResultObject } from "App/core/builder"
+import { AppError } from "App/core/errors"
+import { Endpoint, Method } from "App/device/constants"
+import { DeviceInfo } from "App/device-info/dto"
+import { DeviceInfoPresenter } from "App/device-info/presenters"
+
+// DEPRECATED
+import DeviceService from "App/__deprecated__/backend/device-service"
+
+export class DeviceInfoService {
+  constructor(private deviceService: DeviceService) {}
+
+  public async getDeviceInfo(): Promise<ResultObject<DeviceInfo>> {
+    try {
+      const response = await this.deviceService.request({
+        endpoint: Endpoint.DeviceInfo,
+        method: Method.Get,
+      })
+
+      if (!response.data) {
+        return Result.failed(new AppError("", ""))
```

Comment:
Just wondering - is it ok that in some files (e.g. here or in `packages/app/src/device/services/device.service.ts`) `AppError` is returned with empty strings?

Reply:
I will fill in those errors in the next iterations